src/sentry/lint/engine.py

```diff
@@ -322,8 +322,9 @@ def run_formatter(cmd, file_list, prompt_on_changes=True):
                 print('\033[1m' + 'Stage this patch and continue? [Y/n] ' + '\033[0m')  # noqa: B314
                 if fp.readline().strip().lower() != 'y':
                     print(  # noqa: B314
-                        '[sentry.lint] Aborted! Changes have been applied but not staged.', file=sys.stderr)
+                        '[sentry.lint] Changes have been applied but not staged.', file=sys.stderr)
```

Comment:
I don't think this should be printed to stderr.

Reply:
They already are.

Reply:
Isn't the message a bit inaccurate in that there may not have been changes done by the linter? (e.g. you started with a un-staged changes)

Reply:
Yes true, I'll change it to 'Unstaged changes have not been staged."